command/genesis/params.go

```diff
@@ -384,6 +402,46 @@ func (p *genesisParams) predeployStakingSC() (*chain.GenesisAccount, error) {
 	return stakingAccount, nil
 }
 
+func (p *genesisParams) extractTokenParams() error {
```

Comment:
Consider moving it to `polybft_params.go`, so it would be easier for us when we start removing IBFT code :slightly_smiling_face:

Reply:
I need it at the `genesisParams` because I want to do validation before executing command, and not while creating `polyBft_config`.